Lucca.Logs.AspnetCore/LuccaLoggerExtensions.cs

```diff
@@ -6,7 +6,7 @@
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Logging;
 using StackExchange.Exceptional;
-#if NETCOREAPP3_1
+#if NET6_0
```

Comment:
Not sure if it would be useful in that case. If it does, be sure to replace all the other occurrences as well. 🙂 

```suggestion
#if NET6_0_OR_GREATER
```

Reply:
indeed